ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/patch-panel.j2

```diff
@@ -0,0 +1,29 @@
+{% if patch_panel is arista.avd.defined %}
+
+## Patch Panel
+
+### Patch Panel Summary
+
+| Patch Name | Connector 1 | Connector 2 | Enabled |
+| ---------- | ----------- | ----------- | ------- |
+{% for patch in patch_panel.patches %}
```

Comment:
make sure to sort the same way as in config.
Make sure to test all variables or use default or natural_sort

Reply:
Columns sorted in same way as configuration. Missing checks for variables added.